Sources/Apollo/ApolloClient.swift

```diff
@@ -138,7 +138,6 @@ extension ApolloClient: ApolloClientProtocol {
     }
   }
   
-  @discardableResult
```

Comment:
Our [documentation](https://www.apollographql.com/docs/ios/fetching/subscriptions#cancelling-a-subscription) states that subscriptions must be cancelled but through `@discardableResult` users were not forced to hold on to the result of the subscription request and therefore had no way to cancel it.

Reply:
I'm all for this, but this is technically an API breaking change. It seems to be a pretty significant error though, so I'm okay with this!

Reply:
Correct, leans more towards a "bug fix" IMO.